server/docker/Dockerfile.android.ndk25-env

```diff
@@ -99,6 +99,15 @@ RUN --mount=type=secret,id=DM_PACKAGES_URL,required=true \
     aapt v  && \
     aapt2 version
 
+# download and install R8 utility separatly. It's done in that way because
+# R8/D8 received fixes from time to time which not populated with build tools
+# It's ok that R8.jar downloaded but saved to D8.jar. Can't find D8 as separate utility.
+# R8 contains all classes of D8.
+ENV R8_VERSION=8.7.0-dev
+RUN \
+    wget -O ${ANDROID_SDK_BUILD_TOOLS_PATH_34}/lib/d8.jar https://storage.googleapis.com/r8-releases/raw/${R8_VERSION}/r8.jar && \
+    wget -O ${ANDROID_SDK_BUILD_TOOLS_PATH_33}/lib/d8.jar https://storage.googleapis.com/r8-releases/raw/${R8_VERSION}/r8.jar
```

Comment:
Updated both because during the build it picks from build tools 33.0.1. It happened because ProcessExecutor doesn't respect PATH variable when start the process.